src/InvestmentManager.sol

```diff
@@ -635,36 +661,34 @@ contract InvestmentManager is Auth {
         }
     }
 
-    /// @dev convert decimals of the value into the pool decimals
-    function _toPoolDecimals(uint128 _value, uint8 decimals, address liquidityPool)
+    /// @dev when converting currency to tranche token amounts using the price,
+    /// all values are normalized to PRICE_DECIMALS
+    function _toPriceDecimals(uint128 _value, uint8 decimals, address liquidityPool)
         internal
         view
-        returns (uint128 value)
+        returns (uint256 value)
     {
-        (uint8 maxDecimals,,) = _getPoolDecimals(liquidityPool);
-        if (maxDecimals == decimals) return _value;
-        return _toUint128(_value * 10 ** (maxDecimals - decimals));
+        if (PRICE_DECIMALS == decimals) return uint256(_value);
```

Comment:
This line is unneeded isn't it? `_value * 10 ** 0` returns `_value` anyway. Or is there some weird solidity edge-case here I'm not seeing?

Reply:
It just is more efficient to skip the pow multiplication below if we don't need it.